app/components/Header/tests/A.test.js

```diff
@@ -1,27 +1,12 @@
 import React from 'react';
-import { mount, render } from 'enzyme';
+import { shallow } from 'enzyme';
 
 import A from '../A';
 
-describe('<A />', () => {
-  it('should render an <a> tag', () => {
-    const renderedComponent = render(<A />);
-    expect(renderedComponent.find('a').length).toEqual(1);
-  });
-
-  it('should have a className attribute', () => {
-    const renderedComponent = mount(<A />);
-    expect(renderedComponent.find('a').prop('className')).toBeDefined();
-  });
-
-  it('should adopt a valid attribute', () => {
-    const id = 'test';
-    const renderedComponent = mount(<A id={id} />);
-    expect(renderedComponent.find('a').prop('id')).toEqual(id);
-  });
+const wrapper = shallow(<A />);
```

Comment:
Why are we removing many of the tests that use mount and render throughout the example app?